packages/core-api/src/app/App.tsx

```diff
@@ -141,18 +147,32 @@ export class PrivateAppImpl implements BackstageApp {
   }
 
   getProvider(): ComponentType<{}> {
-    const appApis = ApiRegistry.from([
-      [appThemeApiRef, AppThemeSelector.createWithStorage(this.themes)],
-    ]);
-    const apis = new ApiAggregator(this.apis, appApis);
-
-    const Provider: FC<{}> = ({ children }) => (
-      <ApiProvider apis={apis}>
-        <AppContextProvider app={this}>
-          <AppThemeProvider>{children}</AppThemeProvider>
-        </AppContextProvider>
-      </ApiProvider>
-    );
+    const Provider: FC<{}> = ({ children }) => {
+      const config = useAsync(this.configLoader);
+      if (config.loading) {
+        return null;
```

Comment:
Progress? That only appears after some time

Reply:
Ah yeah this should at least have a TODO to add a Progress component